tools/metadata_merger/metadata_merger.xml

```diff
@@ -0,0 +1,38 @@
+<tool id="metadata_merger" name="Metadata Merger" version="0.0.1" python_template_version="3.5">
+    <description>merges metadata with features table</description>
+    <requirements>
+        <requirement type="package" version="3.8.5">python</requirement>
+        <requirement type="package" version="1.2.1">pandas</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        python $__tool_directory__/metadata_merger.py --dataframe "$features_table" --extension "${features_table.ext}" --metadata "$metadata_table" --output "$merged_data"
```

Comment:
Please make sure the needed variables are single quoted, so they don't get expanded in the shell. (from [IUC standards](https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html#command-formatting))

> All Cheetah variables for text parameters, input and output files must be single-quoted, e.g. '${var_name}'.